.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "modules/mavlink_vehicles"]
 	path = modules/mavlink_vehicles
-	url = https://github.com/01org/mavlink-vehicles.git
+	url = https://github.com/guiccbr/mavlink-vehicles.git
```

Comment:
Just temporarily. I expect [this patch](https://github.com/01org/mavlink-vehicles/pull/5) to be merged before this one. When that happens, I'll get rid of this commit and get back to the original address.